BlogTemplate/Pages/AlreadyRegistered.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace BlogTemplate.Pages
+{
+    public class AlreadyRegisteredModel : PageModel
+    {
+        public void OnGet()
+        {
+        }
+    }
+}
```

Comment:
Actually, if this page doesn't have any handlers, we could just get rid of this whole file.